src/main/java/com/nextcloud/android/lib/resources/directediting/DirectEditingCreateFileRemoteOperation.java

```diff
@@ -66,6 +66,7 @@ protected RemoteOperationResult run(OwnCloudClient client) {
 
         try {
             postMethod = new PostMethod(client.getBaseUri() + DIRECT_ENDPOINT + JSON_FORMAT);
+            postMethod.getParams().setContentCharset("utf-8");
```

Comment:
@juliushaertl is this the correct way to handle utf post bodies?

Reply:
Yes, that should do the trick :wink:

Reply:
Can you imagine of any circumstance that prevents us to put this as default in every post method?